### PR TITLE
Add support for DBR multisite tests with tenanted users

### PR DESCRIPTION
**Pull request by mkasturi18**
Signed-off-by: MadhaviKasturi <mkasturi@redhat.com>

# Description
Add support to test multisite DBR tests on Greenfield(fresh) deployments with tenanted users. Currently we have the scenario covered only in upgrade suites.

Polarion testcase - https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574433

Pass logs:
RHCS 5.3 -  http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-GYHODZ
RHCS 6.0 - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-JOZZKX

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: pipeline/metadata/6.0.yaml [hunk 1]
```diff
@@ -949,3 +949,19 @@ suites:
...
       - stage-4
+
+  - name: "test-rgw-ms-dynamic-bucket-resharding"
+    suite: "suites/pacific/rgw/tier-2_rgw_ms-dynamic-bucket-resharding.yaml"
+    global-conf: "conf/pacific/rgw/rgw_multisite.yaml"
+    platform: "rhel-9"
+    rhbuild: "6.0"
+    inventory:
+      openstack: "conf/inventory/rhel-8-latest.yaml"
+      ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
```
Comment (udaysk23) [inline pipeline/metadata/6.0.yaml:960]: @mkasturi18 Could you please confirm the rehel version here

**Comment by udaysk23** [comment]
@mkasturi18 It looks like we have incorrect link of 5.3 run
Could you please cross check that as well

File: pipeline/metadata/6.0.yaml (hunk 1 above)
Comment (mkasturi18) [reply pipeline/metadata/6.0.yaml]: @udaysk23 , changed the rhel version accordingly

**Comment by mkasturi18** [comment]
@udaysk23 , the PR is to add support for multisite DBR with tenanted users, the log is here http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-GYHODZ/create_tenanted_user_0.log

**Review by udaysk23 — APPROVED** [review]

**Review by ckulal — APPROVED** [review]

**Pull request merged**